TACC/Core-Portal
# task/FP-1635: 737 - Implement final Allocations UI

## Overview: ##
Update Allocations UI to match final design

## Related Jira tickets: ##

* [FP-1635](https://jira.tacc.utexas.edu/browse/FP-1635)

## Summary of Changes: ##
- Combined view team and manage team modals and enabled toggling between views with tabs
- Added styling for said tabs
- Updated view team button on Allocations page to be more pronounced

## Testing Steps: ##
1. Go to [Allocations page](https://cep.dev/workbench/allocations/approved)
2. Confirm that 'Manage Allocations' button in top right of window now reads 'Request New Allocation'
3. Confirm that View Team button is now a rectangular, grey, secondary-styled button
4. Open View Team modal (with 'View Team' button') on a project where you are a PI or delegate
5. Confirm content of modal matches design (tabs, each modal window, etc) (see Notes for design link)

## UI Photos:
<img width="1401" alt="image" src="https://user-images.githubusercontent.com/43251554/167715860-e47946ea-51a8-42ec-b9b1-62c2fc9709d2.png">
<img width="802" alt="image" src="https://user-images.githubusercontent.com/43251554/167715952-3eb7f179-4fb2-4a95-ad0b-830d150ba256.png">
<img width="802" alt="image" src="https://user-images.githubusercontent.com/43251554/167716034-a39ea040-d0a9-42d8-b680-3fb54230c917.png">


## Notes: ##
[Design (slides 9 - 12) ](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/9285756d-03e0-4be2-8926-3c8db95f6947/specs/)
- At this time, the proposed design to change manage team search result display will not be implemented (see [this Slack thread](https://tacc-team.slack.com/archives/C02K9CB9ZAA/p1651590715314609))
- New Allocations tabs are going to be spun off to another task
